### PR TITLE
Harmonization of gui

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: controllers/statisticsController.ts [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
+    psychologistCount,
+    patientsCount,
+  ] = await Promise.all([university, appointments, psychologist, patients]);
   res.json([
     {
       label: 'Universités partenaires',
       value: universityCount[0].count,
     },
+    {
+      label: 'Nombre de séances réalisées',
```
Comment (sandrica89) [inline controllers/statisticsController.ts:25]:
```suggestion
      label: 'Séances réalisées',
```
to use same wording as the other stats

File: frontend/cypress/integration/landing.spec.js [hunk 2]
```diff
@@ -17,6 +17,8 @@ describe('Landing Page Test', () => {
     cy.wait('@statistics');
     cy.get('[data-test-id="statistic-Universités partenaires"]').should('exist');
     cy.get('[data-test-id="statistic-Universités partenaires-value"]').should('have.text', 11);
+    cy.get('[data-test-id="statistic-Nombre de séances réalisées"]').should('exist');
+    cy.get('[data-test-id="statistic-Nombre de séances réalisées-value"]').should('have.text', 156);
```
Comment (sandrica89) [inline frontend/cypress/integration/landing.spec.js:21]:
```suggestion
    cy.get('[data-test-id="statistic-Séances réalisées"]').should('exist');
    cy.get('[data-test-id="statistic-Séances réalisées-value"]').should('have.text', 156);
```
(if previous suggestion was applied)

File: test/helper/clean.ts [hunk 3]
```diff
@@ -47,20 +53,21 @@ const getOnePsy = (
     archived,
     state,
     adeli: `${getRandomInt()}829302942`,
-    address: `${getRandomInt()} SOLA 66110 MONTBOLO`,
+    address: `${getRandomInt()} ${addresses[Math.floor(Math.random() * 100) % 3]}`,
```
Comment (sandrica89) [inline test/helper/clean.ts:56]: To generate random values that seems real, we could use [faker](https://github.com/Marak/Faker.js)

**Review by sandrica89 — CHANGES_REQUESTED** [review]
Responsive issues:
PR314-01
PR314-02
PR314-03

File: test/helper/clean.ts (hunk 3 above)
Comment (desoindx) [reply test/helper/clean.ts:56]: https://trello.com/c/qlmb3r34/159-use-faker-to-generate-mock-data

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**